kuma/wiki/jinja2/wiki/react_document.html

```diff
@@ -93,7 +93,7 @@
 
   {#{% include "includes/a11y-nav.html" %}#}
 
-  {{ render_react_app(document_api_data)|safe }}
+  {{ render_react_app(document_api_data, request_locale)|safe }}
```

Comment:
I can't find where `request_locale` is defined.

Reply:
It seems to me that `request_locale` is not needed anyway since `requestLocale` is already in the `document_api_data`, so it seems this can remain:
```
{{ render_react_app(document_api_data)|safe }}
```

Reply:
You're right. I'll take that second argument out.  (I tried doing this two different ways... This is a leftover from the first way. Thank you for catching it.